packages/fxa-content-server/app/scripts/views/sign_in_unblock.js

```diff
@@ -64,6 +130,13 @@ const View = FormView.extend({
       // re-throw, it'll be displayed at a lower level.
       throw error;
     }
+
+    if(AuthErrors.is(error, 'INVALID_UNBLOCK_CODE')) {
+      this.showValidationError(this.getElementValue('#unblock_code'),error);
```

Comment:
The showValidationError function needs to be passed an element selector and error to display the tooltip.